trunk/SUAVE/Attributes/Solids/Acrylic.py

```diff
@@ -0,0 +1,31 @@
+## @ingroup Attributs-Solids
+
+# Acrylic.py
+#
+# Created: Jul 2017, J. Smart
+
+#-------------------------------------------------------------------------------
+# Imports
+#-------------------------------------------------------------------------------
+
+from Solid import solid
+from SUAVE.Core import Data, Units
+
+#-------------------------------------------------------------------------------
+# Acrylic Solid Class
+#-------------------------------------------------------------------------------
+
+class acrylic(solid):
+
+    """ Physical Constants Specific to Polymethyl Methacrylate"""
+
+    def __defaults__(self):
+
+        self.ultimate_tensile_strength        = 75e6          *Units.Pa
```

Comment:
you have extra space, also put a space between the times

Reply:
Spacing adjustments made.